src/screens/Auth/SignIn/signIn.jsx

```diff
@@ -0,0 +1,46 @@
+import {
+  TouchableOpacity,
+  SafeAreaView,
+  Text,
+  TextInput,
+  View,
+  Image,
+} from 'react-native';
+import React, {useState} from 'react';
+import {styles} from './styles';
+import {FormField} from '../../../components/FormField/formField';
+
+const guide_explain_tourist = require('../../../assets/images/guide_explain_tourist.jpg');
+
+export const SignIn = ({navigation}) => {
+  const [form, setForm] = useState({
+    email: ' ',
+    password: '',
+  });
+
+  const handleSignIn = () => {};
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <View style={styles.imageContainer}>
+        <Image source={guide_explain_tourist} style={styles.image} />
+      </View>
+      <Text style={styles.welcomeText}>Welcome to Wanderly</Text>
+      <Text style={styles.loginText}>Login to your account</Text>
```

Comment:
let's create a separate constants file and move constant text there instead of hardcoding it in components

Reply:
Sure Awais Bhai, I will definitely look into it.